test/videojs-contrib-hls.test.js

```diff
@@ -2769,6 +2775,13 @@ QUnit.test('blacklists playlist if key requests fail', function(assert) {
   hls.mediaSource.trigger('sourceopen');
   this.requests.shift()
     .respond(200, null,
+              '#EXTM3U\n' +
```

Comment:
Because it has to have multiple playlists then the playlist which key requests fail can be blacklisted. It seems I forgot to change the media.m3u8 to master.m3u8.